### PR TITLE
[FIX] Do not fetch project donors outside AW

### PR DESCRIPTION
**Pull request by aziolek**

### CODE REVIEW

File: client/src/hooks/queries/donors/useProjectsDonors.ts [hunk 1]
```diff
@@ -21,16 +24,9 @@ export default function useProjectsDonors(epoch?: number): {
...
-      queryFn: () =>
-        apiGetProjectDonors(
-          projectAddress,
-          epoch || (isDecisionWindowOpen ? currentEpoch! - 1 : currentEpoch!),
-        ),
-      queryKey: QUERY_KEYS.projectDonors(
-        projectAddress,
-        epoch || (isDecisionWindowOpen ? currentEpoch! - 1 : currentEpoch!),
-      ),
+      enabled: !!projectsAddresses && isEnabled(isDecisionWindowOpen, epoch),
```
Comment (jmikolajczyk) [inline client/src/hooks/queries/donors/useProjectsDonors.ts:27]: I would create variable `isEnabled` in hook scope instead of method outside the hook.

Comment (aziolek) [reply client/src/hooks/queries/donors/useProjectsDonors.ts:27]: But it's exported and used in `client/src/hooks/queries/useProjectsIpfsWithRewards.ts`.

Comment (aziolek) [reply client/src/hooks/queries/donors/useProjectsDonors.ts]: Done.

**Review by jmikolajczyk — APPROVED** [review]

**Comment by aziolek** [comment]
Waiting for https://chat.wildland.dev/wildland/pl/tdcfqiuhutb5pbu3iobwbwsg1h to resolve.

**Review by jmikolajczyk — APPROVED** [review]

**Pull request merged**